examples/streaming-pwa/js/index.js

```diff
@@ -5,6 +5,24 @@ const searchField = document.querySelector("#search-field")
 const searchButton = document.querySelector("#search-button")
 const resultsContainer = document.querySelector("#results")
 
+// Quit if the browser doesn't support WritableStream
+if( window.WritableStream === undefined ){
+  
+  // oh no
+  console.log("Oops! Your current browser doesn't support streaming :(")
+
+  // Alert the user via dangerously set inner html
+  resultsContainer.innerHTML = `
```

Comment:
I would actually put this in the `index.html` and then just add a CSS class `show` with JS to the querySelected element, for encapsulation. Do you think that makes sense?

Reply:
Just leave a `<div class="hidden error">...</div>` in the index file, then get the component by the .error identifier and set className="error" (no need for a special show class, just a hide {display: none; })? That sounds reasonable

Reply:
Or even `document.querySelector('.error').classList.remove('hidden')` to show it.

Reply:
Ideally the querySelector call lives at the top with the other querySelectors and we simply do `error.classList.remove('hidden') // <- show the error`, following the `Basically,` style of simple communication.

Reply:
Sounds good

Reply:
Thanks for the discussion! Excited for the next commit. 😉